packages/core/exports/vue.ts

```diff
@@ -0,0 +1,58 @@
+import { AccountController } from '../src/controllers/AccountController.js'
+import { CoreHelperUtil } from '../src/utils/CoreHelperUtil.js'
+import { ChainController } from '../src/controllers/ChainController.js'
+import type { CaipNetwork, CaipNetworkId } from '@reown/appkit-common'
+import { ref } from 'valtio/vanilla'
+import { onUnmounted } from 'vue'
+
+// -- Hooks ------------------------------------------------------------
+export function useAppKitNetwork(): {
```

Comment:
This might be a separate PR. We don't properly provide hooks for Vue. This is fixing that